openmetadata-ui/src/main/resources/ui/src/utils/SvgUtils.tsx

```diff
@@ -25,6 +25,7 @@ import IconAddReaction from '../assets/svg/add-reaction-emoji.svg';
 import IconAdmin from '../assets/svg/admin.svg';
 import IconAlertBell from '../assets/svg/alert-bell.svg';
 import IconAllActivity from '../assets/svg/all-activity.svg';
+import IconAnnouncementsBasicPrimary from '../assets/svg/announcements-basic-primary.svg';
```

Comment:
@Ashish8689, Don't we have this icon already?

Reply:
@Sachin-chaurasiya  yes, we have it in black color and this icon is a primary color.